### PR TITLE
Sample_PullRequest_check

### PR DESCRIPTION
**Pull request by VinothTesting7**
pulling the request for first time

### CODE REVIEW

**Comment by Dayanandmdr** [comment]
hi vinoth plz check code

**Comment by VinothTesting7** [comment]
@Dayanandmdr Hi daya, can u plz check that committed code and  make correct changes to it and merge same to the master branch

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # Hello_World
 Sample Github learning
+
+Hi ,
```
Comment (Dayanandmdr) [inline README.md]: INSTED of hi put hello

Comment (VinothTesting7) [reply README.md]: ok done

**Pull request merged**